src/org/ods/orchestration/service/JiraService.groovy

```diff
@@ -793,4 +799,79 @@ class JiraService {
         }
         return new JsonSlurperClassic().parseText(response.getBody())
     }
+
+    protected List loadJiraSecurityVulnerabilityIssues(String issueSummary, String fixVersion,
```

Comment:
This is not generic enough to be in JiraService. You can move it to the JiraUseCase.

Reply:
Done.

Reply:
Fine now